.github/workflows/publish-provider-family.yml

```diff
@@ -0,0 +1,175 @@
+name: Publish Provider Family
+
+on:
+  workflow_dispatch:
+    inputs:
+      repository:
+        description: 'Official provider family repository to build and publish'
+        default: 'crossplane-contrib/provider-upjet-aws'
```

Comment:
nit: What do you think about making this parameter required and removing the default? For now, we foresee using it only in AWS (since no disk issues have been observed in other providers), but we may also need bigger runners for different providers. Since this workflow is centralized and has release capabilities for any provider, we might consider not having a default for the parameter.

Reply:
Done. Moved `crossplane-contrib/provider-upjet-aws` to the description as an example.